modules/gapi/misc/python/pyopencv_gapi.hpp

```diff
@@ -43,3 +92,44 @@ static PyObject* pyopencv_cv_GOut(PyObject* , PyObject* py_args, PyObject* kw)
 {
     return extract_proto_args<GProtoOutputArgs>(py_args, kw);
 }
+
+static PyObject* pyopencv_cv_gin(PyObject* , PyObject* py_args, PyObject* kw)
+{
+    using namespace cv;
+
+    GRunArgs args;
+    Py_ssize_t size = PyTuple_Size(py_args);
+    for (int i = 0; i < size; ++i) {
+        PyObject* item = PyTuple_GetItem(py_args, i);
+        if (PyTuple_Check(item)) {
+            cv::Scalar s;
+            if (pyopencv_to(item, s, ArgInfo("scalar", true)))
+            {
+                args.emplace_back(s);
+            }
+            else
+            {
+                PyErr_SetString(PyExc_TypeError, "Failed convert tuple to cv::Scalar");
+                return NULL;
+            }
+        } else if (PyArray_Check(item)) {
+            cv::Mat m;
```

Comment:
Please choose one style of openning `{` (same line or a new one)

Reply:
Fixed